src/main/webapp/constants.js

```diff
@@ -14,8 +14,16 @@ limitations under the License.*/
 // DOM elements
 const PIE_CHART= "piechart";
 const FILE_NAME = "fileName";
+const SLIDE_1_CONTAINER = "slide1Container";
+const SLIDE_2_CONTAINER = "slide2Container";
+const SLIDE_1 = "slide1";
+const SLIDE_2 = "slide2";
+const FILE_NOT_FOUND = "File Not Found";
```

Comment:
Please create a separate constant section for this.
It isn't DOM Element. It's an HTML content.